src/modules/commentsPopup.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-underscore-dangle */
+const openCommentWindow = (movie) => {
+  const CommentsPopup = document.querySelector('.comments-popup');
+  CommentsPopup.style.display = 'flex';
+
+  CommentsPopup.innerHTML = `<div class="top">
+                // eslint-disable-next-line no-underscore-dangle
```

Comment:
Please consider removing that commented line, it displays on the popup. Look at the attached image below
![noel2](https://github.com/NoelLincoln/movie-safari/assets/43038485/4026625d-e552-4460-b0be-02ba9b8be7ab)